Cabal/src/Distribution/Simple/HaskellSuite.hs

```diff
@@ -184,8 +186,8 @@ buildLib verbosity pkg_descr lbi lib clbi = do
       language = fromMaybe Haskell98 (defaultLanguage bi)
       progdb = withPrograms lbi
       pkgid = packageId pkg_descr
-      i = interpretSymbolicPathLBI lbi -- See Note [Symbolic paths] in Distribution.Utils.Path
-  runDbProgram verbosity haskellSuiteProgram progdb $
+      i = interpretSymbolicPathCWD -- See Note [Symbolic paths] in Distribution.Utils.Path
```

Comment:
Could you rename `i` to `u`? That's the convention I've been using when we are using `interpretSymbolicPathCWD`.

Reply:
done.